kafka/docker/template/jmxtrans-agent.xml

```diff
@@ -0,0 +1,110 @@
+<!--
+  ~ Copyright 2018 Expedia Group, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<jmxtrans-agent>
+    <queries>
+
+        <!-- JVM -->
+        <query objectName="java.lang:type=Memory" attribute="HeapMemoryUsage" key="used"
```

Comment:
Don't know what any of these are for. Just copied from https://github.com/ExpediaDotCom/haystack-trends/blob/ccf14b1c26a0482cb0dcc93f1acd2799d477a189/timeseries-aggregator/build/docker/jmxtrans-agent.xml.